java/java_test.go

```diff
@@ -376,6 +376,44 @@ func testJavaWithBuilder(builder string) func(*testing.T, spec.G, spec.S) {
 				})
 			})
 
+			context("app uses opentelemetry", func() {
+				it("builds successfully", func() {
+					var err error
+					source, err = occam.Source(filepath.Join("../java", "opentelemetry"))
+					Expect(err).NotTo(HaveOccurred())
+
+					var logs fmt.Stringer
+					image, logs, err = pack.Build.
+						WithPullPolicy("never").
+						WithEnv(map[string]string{
+							"BP_OPENTELEMETRY_ENABLED": "true",
+						}).
+						WithBuilder(builder).
+						WithBuildpacks(
+							"paketo-buildpacks/java",
+							"gcr.io/paketo-buildpacks/opentelemetry",
+						).
+						WithVolumes(fmt.Sprintf("%s/.gradle:/home/cnb/.gradle:rw", home)).
+						WithGID("121").
```

Comment:
```suggestion
						WithGID("123").
```

This should be `123` based on recent changes in Github Actions.

Reply:
Done